.rubocop.yml

```diff
@@ -34,6 +34,12 @@ Metrics/BlockLength:
   Exclude:
     - "spec/**/*"
 
+Style/TrailingCommaInArrayLiteral:
+  EnforcedStyleForMultiline: comma
```

Comment:
If there is a option to decide when is "good" to be multiline, for me, it's ok. Little arrays I don't know if makes sense to be multiline. Example:

```Ruby
# good
a = [ 1 , 2 ]

# good
a = [
  1,
  2,
  3,
  4,
  5
]

# bad
a = [ 1, 2, 3, 4, 5, 6, 7 ]


```

Reply:
This cop just looks at arrays and hashes that are multi line. If they are, it expects that they have a trailing comma. It can't look at one liners, so it can't tell if it's too big or not...
there's something like that but it's for methods https://github.com/rubocop-hq/ruby-style-guide#too-many-params